### PR TITLE
Issue incorrect exception message

### PR DESCRIPTION
**Pull request by mrj001**
Incorporated unit test from Issue #34413 
Corrected the exception message being thrown.  Since it is the fractionDigits facet which is illegal, this is now the one mentioned in the exception message.

### CODE REVIEW

**Review by buyaa-n — APPROVED** [review]
@mrj001 thank you for your contribution, left few NIT, other than that LGTM

**Comment by mrj001** [comment]
@buyaa-n 
I take it that the NITs can remain? 

As there's one failure above, what do I need to do to finish this pull request?

**Comment by buyaa-n** [comment]
> I take it that the NITs can remain?

Not really, it is not big issue mostly styling but preferred to be fixed unless you have a point not to do that or you are going change it with a another subsequent PR 
> As there's one failure above, what do I need to do to finish this pull request?

I pushed the failed build to rerun, but it might fail again. Also when you commit a change it would do new full build which most likely fix the failed build.

File: src/libraries/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_Compile.cs [hunk 1]
```diff
@@ -149,5 +149,33 @@ public void TFS_470021()
                 Assert.Equal(0, errorCount);
             }
         }
+
+
```
[the remark this answers is not in the document]
Comment (mrj001) [reply src/libraries/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_Compile.cs:153]: Removed

File: src/libraries/System.Private.Xml/src/Resources/Strings.resx [hunk 2]
```diff
@@ -3452,4 +3453,7 @@ Usage: dotnet {0} [--assembly &lt;assembly file path&gt;] [--type &lt;type name&
   <data name="CompilingScriptsNotSupported" xml:space="preserve">
     <value>Compiling JScript/CSharp scripts is not supported</value>
   </data>
+  <data name="Sch_FractionDigitsMismatch" xml:space="preserve">
+    <value>It is an error if the derived 'fractionDigits' facet value is greater than the parent 'fractionDigits' facet value.</value>
+  </data>
```
[the remark this answers is not in the document]
Comment (mrj001) [reply src/libraries/System.Private.Xml/src/Resources/Strings.resx]: Addressed and re-pushed the branch.

[the remark this answers is not in the document]
Comment (buyaa-n) [reply src/libraries/System.Private.Xml/src/Resources/Strings.resx]: Great, thank you @mrj001 merging

**Pull request merged**